.github/workflows/build-deps.yml

```diff
@@ -0,0 +1,196 @@
+name: Build emsdk dependencies
+
+on:
+  workflow_dispatch:
+    inputs:
+      llvm_build:
+        description: 'LLVM Webassembly arm64'
+        required: false
+        default: false
+        type: boolean
+      llvm_build_cross_compilation:
+        description: 'LLVM Webassembly arm64 (cross compilation)'
+        required: false
+        default: false
+        type: boolean
+      emscripten_build:
+        description: 'Emscripten'
+        required: false
+        default: false
+        type: boolean
+      nodejs_build:
+        description: 'Node.js arm64'
+        required: false
+        default: false
+        type: boolean
+      emsdk_build:
+        description: 'emsdk arm64'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build-llvm-webassembly-arm64:
+    if: ${{ inputs.llvm_build }} ||  ${{ inputs.emsdk_build }}
```

Comment:
Why is LLVM build enabled only when emsdk is enabled? I guess it makes sense to test LLVM build even if the final emsdk package is not being produced.

Reply:
it is used in emsdk_buld needs: however I am not sure how condition will behave in this case when it is mentioned as needed however feature flag was not selected

Reply:
If dependent job was not triggered, the dependee job wont't be triggerd by default. I guess that's a desired behavior in this case. There is a trick how to proceed with running dependee job when dependend one is not enabled that can be seen here https://github.com/Windows-on-ARM-Experiments/emsdk/blob/emsdk-build-from-sources/.github/workflows/build-with-emsdk.yml#L185

Reply:
Ok, I will check how it works

Reply:
Actually the condition always evaluates to true, the correct syntax is:  
\`\`\`  
if: ${{ inputs.llvm\_build || inputs.emsdk\_build }}  
\`\`\`

Reply:
Actually, this always evaluate to true. The correct syntax is:

```
if: ${{ inputs.llvm_build || inputs.emsdk_build }}
```